configure.ac

```diff
@@ -97,4 +162,6 @@ AC_CONFIG_FILES([Makefile
 
 AC_OUTPUT
 
-AC_MSG_RESULT(DONE: Configuration done. Run make/gmake install to install CFEngine Community Masterfiles.)
+AC_MSG_RESULT()
+AC_MSG_RESULT(DONE: Configuration done. Run "make install" to install CFEngine Masterfiles.)
+AC_MSG_RESULT()
```

Comment:
Yeah but distributors of ports systems are well aware of it, no need to remind it every time.